cluster/events.go

```diff
@@ -0,0 +1,20 @@
+package cluster
+
+import "github.com/docker/docker/api/types/events"
+
+// Event is exported
+type Event struct {
+	events.Message
+	Engine *Engine `json:"-"`
+}
+
+// EventHandler is exported
+type EventHandler interface {
+	Handle(*Event) error
```

Comment:
Personally, I like to avoid returning an error when one is never expected to occur, but I don't have a strong preference here.